Assets/Scripts/HealthUI.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class HealthUI : MonoBehaviour {
```

Comment:
Pretty great overall, but because crafting is an UI we already have a canvas, you can add it in there instead of making a new canvas. (Or maybe don't, because it turns out that working in the same scene is deadly when merging in unity, so your changes could need to be thrown away if we change anything in the scene by the time you want to merge this PR... it's a hassle)